lms/djangoapps/onboarding_survey/forms.py

```diff
@@ -47,6 +55,28 @@ def clean_country_of_employment(self):
                 )
         return self.cleaned_data['country_of_employment']
 
+    def clean_country_of_residence(self):
```

Comment:
Can we write a common function for the 3 functions below? There is lot of code duplication in them